LIBVmonitor0.11/LIBVmonitor0.11.ino

```diff
@@ -162,31 +194,27 @@ void loop() {
 }
 
 
-void alarmTrigger(bool _al_stat, int _al_num){
-	int _num_count[3];
-	long _time_on[3];
-	long _time_of[3];
-  	if(_al_stat && (al_count<=_num_count[_al_num]){
-  		if(!buzzer_stat && (millis()-al_t >= _time_on[_al_num])){
-  			al_t = millis();
-  			al_stat = true;
-  			al_count++;
-  		}else if(buzzer_stat && (millis()-al_t >= _time_of[_al_num])){
-  			al_t = millis();
-  			al_stat = false;
-  		}
-  	}else al_count=0;
-}
-
 void BEEP(){
```

Comment:
Be careful using this function, as it contains a blocking delay(). If this arduino is supposed to be monitoring the movement of the device, this may cause problems. It might be fine, but just be aware of that.

I'd also recommend passing in the duration of the beep as an argument:
beep(100);